src/components/ClientCard/ClientCard.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { setActiveChat } from '../../store/Chat/actions';
+import './ClientCard.css';
+
+const ClientCard = props => (
+  <li className="ClientCard" >
+    <button onClick={() => props.setActiveChat(props.chatId)}>
```

Comment:
Also by remove eslint rule, I mean make it only spit out a warning instead of an error